Mathlib/CategoryTheory/Monoidal/Internal/FunctorCategory.lean

```diff
@@ -159,6 +122,92 @@ def monFunctorCategoryEquivalence : Mon_ (C ⥤ D) ≌ C ⥤ Mon_ D where
 set_option linter.uppercaseLean3 false in
 #align category_theory.monoidal.Mon_functor_category_equivalence CategoryTheory.Monoidal.monFunctorCategoryEquivalence
 
+namespace ComonFunctorCategoryEquivalence
+
+variable {C D}
+
+/--
+A comonoid object in a functor category induces a functor to the category of comonoid objects.
+-/
+@[simps]
+private def functor_obj' (A : Comon_ (C ⥤ D)) : C ⥤ Comon_ D where
```

Comment:
Should this be private or not? I assume we either both or neither of the `Mon/Comon` versions to be private.

Similarly for all the other `private` decls below

Reply:
Also, `functorObj'` is the proper naming, right?

Reply:
Thanks for the suggestions, I think they are all correct. I've removed `private`.